docker/wsi-worker/generate_mongo_records.js

```diff
@@ -0,0 +1,59 @@
+const MongoClient = require('mongodb').MongoClient;
+const assert = require('assert');
+
+const url = 'mongodb://delphinus-mongodb:27017';
+const dbName = 'knowledgeEnvironment';
+
+var kpmpId = process.argv[2];
+var slideName = process.argv[3];
+var fileUUID = process.argv[4];
+var udpates = [];
```

Comment:
This is misspelled but it doesn't look like it's used anyway.

Reply:
Thanks!